app/assets/javascripts/places.js

```diff
@@ -1,6 +1,30 @@
 // Place all the behaviors and hooks related to the matching controller here.
 // All this logic will automatically be available in application.js.
+$( document ).ready(function() {
+  $(function(){
+    $('[data-toggle="tooltip"]').tooltip();
+  });
 
-$(function () {
-  $('[data-toggle="tooltip"]').tooltip()
+  
+
+  $('#modal-add').on('change', '#pictureInput', function(event){
+    console.log('entra');
+    var files = event.target.files;
+    var image = files[0]
+    var reader = new FileReader();
+    reader.onload = function(file) {
+      var img = new Image();
+      console.log(file);
+      img.src = file.target.result;
+      $('#target').html(img);
+      $('#target').removeClass('photo-frame');
```

Comment:
If you are going to do something with the sale DOM element. I recomend to cache the selector (here you can read more about the topic: http://ttmm.io/tech/selector-caching-jquery/).
So in this case I'd do something like:
```javascript
$target = $('#target')
$target.hml(img)
$target.removeClass('photo-frame');
```